src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsFandomObject.java

```diff
@@ -37,12 +37,31 @@ public AdsFandomObject(WebDriver driver, String testedPage) {
     slots.put(AdsFandomContent.BOTTOM_LEADERBOARD, bottomLeaderboardElement);
     slots.put(AdsFandomContent.TOP_BOXAD, topBoxadElement);
     slots.put(AdsFandomContent.INCONTENT_BOXAD, incontentBoxadElement);
-    slots.put(AdsFandomContent.BOTTOM_BOXAD, bottomBoxadElement);
+  }
+
+  public void triggerOnScrollSlots() {
+    JavascriptExecutor js = driver;
+    js.executeScript("window.scrollTo(0, document.body.scrollHeight);");
+  }
+
+  public long getLineItemId(String slotName) {
+    JavascriptExecutor js = driver;
+    try {
+      return (long) js.executeScript(
+          "var slots = googletag.getSlots(); for (var i = 0; i < slots.length; i++) { " +
+          "if (slots[i].getTargeting('pos').indexOf('" + slotName + "') !== -1) { " +
+          "return slots[i].getResponseInformation().lineItemId;" +
+          "} }"
+      );
+    } catch (WebDriverException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)